pep-0001.txt

```diff
@@ -230,28 +238,40 @@ PEP Review & Resolution
 
 Once the authors have completed a PEP, they may request a review for
 style and consistency from the PEP editors.  However, the content and
-final acceptance of the PEP must be requested of the BDFL, usually via
-an email to the python-dev mailing list.  PEPs are reviewed by the
-BDFL and his chosen consultants, who may accept or reject a PEP or
-send it back to the author(s) for revision.  For a PEP that is
-predetermined to be acceptable (e.g., it is an obvious win as-is
-and/or its implementation has already been checked in) the BDFL may
-also initiate a PEP review, first notifying the PEP author(s) and
+final acceptance of the PEP must be requested of the core developers, usually
```

Comment:
How would this work?  As I read the change, the core developers (not the SC or their delegate) can accept a PEP, but I don't think that process has been worked out yet.  The SC has yet to "establish appropriate decision-making processes for PEPs" as quoted from [PEP 13](https://www.python.org/dev/peps/pep-0013/).

Reply:
I'm trying to word this part in a way that *doesn't* encourage people to send PEP acceptance requests to the steering council mailing list.

As far as how the core devs collectively accept PEPs goes:

1. Core devs collectively elect a Steering Council
2. Steering Council appoints (or chooses not to veto) BDFL-Delegates (who will be core developers themselves in all but the most extraordinary cases)
3. BDFL-Delegates accept or reject individual PEPs

Reply:
Ahah, I have a cunning plan: delete the sentence. It adds no useful information, as it was just restating what the rest of this section explains, but restating it badly :)